README.rst

```diff
@@ -237,9 +237,7 @@ new cassettes. To run new tests, first set up the following environment variable
 
 .. code-block:: bash
 
-    export pycaching_test_username=yourusername
-    export pycaching_test_password=yourpassword
-    python3 setup.py test
+    PYCACHING_TEST_USERNAME="yourusername" PYCACHING_TEST_PASSWORD="yourpassword" python3 setup.py test
 
 Substitute your username for ``yourusername`` and your password for ``yourpassword``. After you
```

Comment:
This needs to be updated too.

Reply:
@tomasbedrich What's the update that needs to happen?